src/compounds/RequestItem/RequestItem.jsx

```diff
@@ -35,11 +41,20 @@ const RequestItem = React.forwardRef(({ createdAt, description, href, img, title
 RequestItem.propTypes = {
   createdAt: PropTypes.string.isRequired,
   description: PropTypes.string.isRequired,
+  href: PropTypes.string,
   id: PropTypes.number,
+  index: PropTypes.number.isRequired,
   img: PropTypes.shape(Image.propTypes).isRequired,
   status: PropTypes.shape(Badge.propTypes).isRequired,
   title: PropTypes.string.isRequired,
   updatedAt: PropTypes.string.isRequired,
 }
 
+RequestItem.defaultProps = {
+  href: '',
```

Comment:
although we aren't manually passing the `href`, it is required and shouldn't have a default props value.

Reply:
<img width="542" alt="image" src="https://user-images.githubusercontent.com/73361970/215607917-55b26888-37ad-4586-8641-43ff061b9073.png">
there is an error in the console that it is undefined. 
since we aren't manually passing it, is there another way to account for this error?

Reply:
if we remove it from the `propTypes` object, does the error go away? if so, I'm fine with that since again, we aren't passing it...it just comes from the React forward ref. (same as the `ref` prop which doesn't appear to be erring?)

Reply:
that causes a lint warning
<img width="825" alt="image" src="https://user-images.githubusercontent.com/73361970/215622470-213f45e5-d1ce-4043-a183-190b81c86fac.png">

but i suppose that is better than the console error. 

the line with forward ref looks like
`React.forwardRef(({ createdAt, description, href, img, index, title, status, updatedAt }, ref)` so ref is outside the props object which is prob why it doesnt trigger the lint warning.

Reply:
if `href` is moved outside of the object does it break the code? if not, does the lint warning go away?

Reply:
yes, moving it outside of the object breaks the code

Reply:
result of console logging href before & after it is moved outside of the object:
<img width="592" alt="image" src="https://user-images.githubusercontent.com/73361970/215817434-0fe6207d-683b-432d-b998-8ecd07f043c3.png">

Reply:
that react forward ref made things difficult. think I figured out a way around the craziness though. take a look at commit sha [02b0216050eac40c29de447a41c7045d9e67eb12](https://github.com/scientist-softserv/webstore-component-library/compare/main...139-refactor-request-list-props). let me know if you have questions. if not, you can merge it in.

Reply:
its been merged in